### PR TITLE
Created ResumeBuilder.js with drag and drop functionality

### PR DESCRIPTION
**Pull request by KesharwaniArpita**
I have Created ResumeBuilder.js with drag and drop functionality only. i have taken assistance from OpenAI as well due to lack of my expertise in JavaScript. Please review my PR and guide me towards solving the issue #8 in this repo and Issue#12 in the python orientation repo.

### CODE REVIEW

File: src/ResumeBuilder.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  const handleDragOver = (e) => {
+    e.preventDefault();
+  };
+
+  const handleDrop = (e) => {
+    e.preventDefault();
+    const sectionId = e.dataTransfer.getData('text/plain');
+    const sectionIndex = sections.findIndex((section) => section.id === sectionId);
+
+    if (sectionIndex !== -1) {
```
Comment (Pradyuman7) [inline src/ResumeBuilder.js]: nit: usually it's easier to understand with early-exit ifs.
```
if (sectionIndex === -1)  { 
    return; 
} 

// other code
```

File: src/ResumeBuilder.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ new_order: sectionOrder }),
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data); // Handle the server response as needed
+      })
+      .catch((error) => {
+        console.error('Error:', error);
```
Comment (Pradyuman7) [inline src/ResumeBuilder.js]: I think we can alert the user here instead of just logging the error

**Comment by KesharwaniArpita** [comment]
I made the suggested changes. I hope it works fine now. And I resolved the convo by mistake, sorry.

**Comment by Pradyuman7** [comment]
> I made the suggested changes. I hope it works fine now. And I resolved the convo by mistake, sorry.

Nice, the convo is meant to be resolved after you fix it, so no worries!

**Review by Pradyuman7 — APPROVED** [review]

**Pull request merged**